packages/authorization-process/test/integration/getClientById.test.ts

```diff
@@ -28,30 +28,16 @@ describe("getClientById", async () => {
     };
     await addOneClient(expectedClient);
 
-    const { client } = await authorizationService.getClientById(
+    const getClientResult = await authorizationService.getClientById(
```

Comment:
I think that this could be better
```suggestion
    const clientResult = await authorizationService.getClientById(
```